go/README.md

```diff
@@ -0,0 +1,73 @@
+# XenServerGo usage examples
+
+## Overview
+
+The following simple examples are included in this repository:
+
+-  `base_test`: Create the basic session, login and logout.
+
+-  `event_test`: Listens for events on a connection and prints each event out 
+    as it is received.
+
+-  `get_all_records_test`: Retrieves the records for all types of objects.
+
+-  `network_test`: Create and destroy a new external network.
+
+-  `pool_test`: Join and eject host of Pool.
+
+-  `session_test`: Set up session over https protocal with or without certificate.
+
+-  `sr_nfs_test`: Creates a shared NFS SR.
+
+-  `sr_test`: Performs various base SR tests, including creating
+    a dummy SR.
+
+-  `vm_create_and_destroy_test`: Create and destroy a VM on the default SR with a network and DVD drive. Do same again with asynchronously call.
```

Comment:
"Do same again with asynchronously call" => "Repeat using asyncrhonous calls".

Same in vm_snapshot_test below.